### PR TITLE
Changesets

### PR DESCRIPTION
**Pull request by orestisrodriguez**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        id: changesets
+        uses: changesets/action@v1
+        with:
+          version: yarn ci:version-packages
+          publish: yarn ci:publish-packages
+
+          title: 'publish changed packages'
+          commit: 'chore: version packages'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (TautvydasGec) [inline .github/workflows/release.yml:40]: is it fine with built in GITHUB_TOKEN cause in lerna pub we used MT_GITHUB_TOKEN cause something didnt work

**Review by TautvydasGec — APPROVED** [review]
looks good just one question

**Comment by SebCodesTheWeb** [comment]
@TautvydasGec so the question was...

**Comment by TautvydasGec** [comment]
Dont you see the comment?

**Pull request closed without merging**